test/WebJobs.Script.Tests/Swagger/SwaggerDocumentManagerTests.cs

```diff
@@ -0,0 +1,307 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web.Http.Routing;
+using System.Web.Http.Routing.Constraints;
+using Microsoft.Azure.WebJobs.Script.Description;
+using Microsoft.Azure.WebJobs.Script.WebHost;
+using Microsoft.Azure.WebJobs.Script.WebHost.Models;
+using Newtonsoft.Json.Linq;
+using WebJobs.Script.Tests;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests
+{
+    public class SwaggerDocumentManagerTests
+    {
+        private ScriptHostConfiguration _scriptConfig;        
```

Comment:
Would it be valuable to have a test comparing against a literal string with an expected swagger document here?